src/main/java/com/github/awwkoala/selenium/page/CheckboxesPage.java

```diff
@@ -0,0 +1,59 @@
+package com.github.awwkoala.selenium.page;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CheckboxesPage extends Page {
+
+  @FindBy(css = "input[type='checkbox']")
+  private List<WebElement> checkboxes;
+
+  public CheckboxesPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public void goToCheckboxesPage() {
```

Comment:
You could think about extracting `goTo` method to abstract Page as an abstract method if your other tasks will be similar.